initial_setup/__init__.py

```diff
@@ -106,7 +107,9 @@ def __init__(self, gui_mode):
         from pyanaconda.core.configuration.base import ConfigurationError
         from pyanaconda.core.configuration.anaconda import conf
         try:
-            conf.set_from_product(get_product_name())
+            conf.set_from_detected_profile(
+                get_os_release_value("ID"),
```

Comment:
Ha, clever! Not sure if the trailing comma is a good idea, though. It tells me there's probably something else, but the helpful hint was overruled by the urge to check if this is actually valid.

Reply:
Fixed :-)